### PR TITLE
Add Seng Leng portfolio and image

### LINKED ISSUE #13: [Seng leng] Update AboutUs

### PR DESCRIPTION
**Pull request by ngsengleng**
### Summary

 Resolves #13

* Update AboutUs with Seng Leng's details
* Add Seng Leng's image and portfolio

### Checks

- [x] Link PR to issue
- [x] Add Reviewer
- [x] Pass CI

### CODE REVIEW

**Comment by simonjulianl** [comment]
Hi, just some general comments about the PR 
* Add the reviewer by the way so one of us can get notified (you can even put our team developers as reviewers)
* Use the keyword Resolves/fixes #issue number, so the kanban board can get updated automatically when your PR is closed 
* Also for the template, if there is nothing to be put, I think can just erase it yup
Thanks man

**Comment by ngsengleng** [comment]
I have fixed the pull request comment. Will take note next time!

File: docs/AboutUs.md [hunk 1]
```diff
@@ -41,10 +41,10 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ### Jean Doe
```
Comment (yyhangz) [inline docs/AboutUs.md:42]: Missed out on the change in name

**Review by yyhangz — CHANGES_REQUESTED** [review]
There is a mistake in your name field under aboutUs, other than that, it looks ok to me

**Pull request merged**